source/NVDAObjects/window/excel.py

```diff
@@ -505,27 +508,51 @@ def navigationHelper(self,direction):
 		cellPosition.Activate()
 		eventHandler.executeEvent('gainFocus',obj)
 
+	@script(
+		gesture=("kb:leftArrow")

```

Comment:
Why you're using a tuple here? In case of a single gesture it is enough to just write `gesture="kb:leftArrow"

Reply:
I was not sure if this would pass the linter. I addressed this now as well.